### ISSUE TITLE
OAuth2TokenValidator examples use non existing method name for creating the NimbusJwtDecoder

### ISSUE THREAD

**Issue by lgraf** [spring-projects/spring-security#6925]
### Summary
The code examples for `OAuth2TokenValidator` use methods that are not exist, for creating a `NimbusJwtDecoder`/`NimbusReactiveJwtDecoder`.

This effects both validator examples (`JwtTimestampValidator` and custom `AudienceValidator`) in both documentations (servlet/reactive).

I have seen that changes to this api are planned (#6010), but i think this change can be helpful in the interim. If you like i provide a PR for adjusting the docs.

### Actual Behavior
The docs use the method `withOidcIssuerLocation(issuerUri)` which does not exists.
```
NimbusJwtDecoder jwtDecoder = (NimbusJwtDecoder)
        JwtDecoders.withOidcIssuerLocation(issuerUri);
```
[Servlet - JwtTimestampValidator example](https://github.com/spring-projects/spring-security/blame/master/docs/manual/src/docs/asciidoc/_includes/servlet/preface/java-configuration.adoc#L724)
[Servlet - AudienceValidator example](https://github.com/spring-projects/spring-security/blame/master/docs/manual/src/docs/asciidoc/_includes/servlet/preface/java-configuration.adoc#L764)

```
NimbusReactiveJwtDecoder jwtDecoder = (NimbusReactiveJwtDecoder)
             ReactiveJwtDecoders.withOidcIssuerLocation(issuerUri);
```
[Reactive - JwtTimestampValidator example](https://github.com/spring-projects/spring-security/blame/master/docs/manual/src/docs/asciidoc/_includes/reactive/oauth2/resource-server.adoc#L344)
[Reactive - AudienceValidator example](https://github.com/spring-projects/spring-security/blame/master/docs/manual/src/docs/asciidoc/_includes/reactive/oauth2/resource-server.adoc#L386)

Sorry for using blame links, but i can't find a way to create a link to a line in file for (rendered) `.adoc` files.

### Expected Behavior
The docs use the method `fromOidcIssuerLocation(issuerUri)`
```
NimbusJwtDecoder jwtDecoder = (NimbusJwtDecoder)
        JwtDecoders.fromOidcIssuerLocation(issuerUri);
```
```
NimbusReactiveJwtDecoder jwtDecoder = (NimbusReactiveJwtDecoder)
             ReactiveJwtDecoders.withOidcIssuerLocation(issuerUri);
```

### Version
5.1.5.RELEASE / master

**Comment by jzheaux** [comment]
@lgraf, thanks for the report! Actually, I'll bet this is just a typo - I agree with your analysis that the examples should use `fromOidcIssuerLocation`.

Would you be interested in submitting a PR to fix that?

**Comment by lgraf** [comment]
@jzheaux i opened a PR where the method names are fixed (#6927).
